kubernetes_state_core/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: Kubernetes State Core
+
+## Overview
+
+Get Kubernetes metrics from [`kube-state-metrics`][1] in real time to:
+
+- Visualize and monitor Kubernetes states.
+- Be notified about Kubernetes failovers and events.
+
+This check supersedes the legacy `kubernetes_state` check.
+Unlike the legacy check, it doesn't require any stand-alone `kube-state-metrics` deployment.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The Kubernetes State Core check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `kubernetes_state_core.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your kubernetes_state_core performance data. See the [sample kubernetes_state_core.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
```

Comment:
IMHO this is not ideal, it's not adapted to containerized environments in general and specifically KSM Core as it runs as a single instance in the cluster. Can we use the same approach we have on this page https://docs.datadoghq.com/integrations/kubernetes_state_core/?tab=helm#configuration ?

Reply:
Based on your remark, I think that the best is just to move all the content of https://docs.datadoghq.com/integrations/kubernetes_state_core/?tab=helm here, except the metrics list that would be in `metadata.csv`. The goal is to have the metrics list not only documented in the doc web site, but also in the app.
WDYT ?
-> https://github.com/DataDog/integrations-core/pull/12552/commits/1a426221cef82509ec6e9c49e3d62de20272db4c

Reply:
Sounds good, thank you!